README.md

```diff
@@ -140,6 +140,7 @@ ENVIRONMENT=development docker-compose build assets
 To watch for changes, run:
 
 ```
+docker-compose build  # only necessary after switching branch or installing new dependencies
```

Comment:
Does this need `-f docker-compose.yml -f docker-compose.watch.yml`?

Reply:
`docker-compose.watch.yml` does not build new images, but only decorates the existing ones with volumes/ports/commands; at the moment, it should not be needed for the build phase

Reply:
Something to remember when that's no longer true.